packages/common/src/visualizations/types/index.ts

```diff
@@ -12,6 +12,15 @@ export enum VizAggregationOptions {
     ANY = 'any',
 }
 
+export enum SortByDirection {
+    ASC = 'asc',
+    DESC = 'desc',
+}
```

Comment:
Unsure about the direction we want to take with types like this one for example. 
In SemanticViewer we have: 
```ts
export enum SemanticLayerSortByDirection {
    ASC = 'ASC',
    DESC = 'DESC',
}
```

Perhaps we want to keep this type above but make them similar in terms of casing for example?